scripts/check.sh

```diff
@@ -150,6 +151,22 @@ if [[ "${ALLOW_DIRTY}" == true && "${COMMIT_METHOD}" == "message" ]]; then
   exit 1
 fi
 
+# When travis clones a repo for building, it uses a shallow clone. After the
+# first commit on a non-master branch, TRAVIS_COMMIT_RANGE is not set, master
+# is not available and we need to compute the START_REVISION from the common
+# ancestor of $TRAVIS_COMMIT and origin/master.
+if [[ "${TEST_ONLY}" == true ]]; then
```

Comment:
`test-only` only indicates we want no changes made, not that it's necessarily a travis run. I think this is unrelated.